src/Registry.ts

```diff
@@ -84,6 +84,20 @@ export function isWidgetBaseConstructor<T extends WidgetBaseInterface>(item: any
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+export interface ESMDefaultWidgetBase<T> {
+	default: Constructor<T>;
+	__esModule: boolean;
+}
+
+export function isWidgetConstructorDefaultExport<T>(item: any): item is ESMDefaultWidgetBase <T> {
```

Comment:
weird space between `ESMDefaultWidgetBase` and the generic

Reply:
won't matter with `prettier`

Reply:
No prettier yet in this repo ;)